A1.md

```diff
@@ -3,14 +3,14 @@ HTTP CONNECT Proxy Support
 * Author(s): Mark D. Roth (roth@google.com), Eric Anderson (ejona@google.com)
 * Approver: a11r
 * Status: Draft
-* Implemented in: (C-core and Java are in progress)
-* Last updated: 2017-01-18
+* Implemented in: C-core (Java is in progress)
+* Last updated: 2017-01-25
 * Discussion at: https://groups.google.com/d/topic/grpc-io/zG7iYpDw6lk/discussion
 
 ## Abstract
 
 Describes what types of TCP-level proxy setups gRPC will support and
-how those setups interact with load balancing.
+how those setups interact with per-call load balancing.
```

Comment:
Server-side L7 load balancing would be the gRPC-level proxy approach described in the "Rationale" section below.  Since this document is mainly concerned with TCP-level proxies, I think it should be clear from context that that's not what's meant here.

That having been said, I am open to using different terminology here, assuming that we can find something that's unambiguous.  As I mentioned earlier, I dislike using the term "client-side" here, because grpclb is fundamentally not a client-side system -- it's a look-aside system.  I think using the term "client-side" may confuse readers who know about grpclb into thinking that we're referring only to round_robin and pick_first, not grpclb.  I think that's more likely to be confusing to readers than the potential implication that we're talking about gRPC-level load balancing.

I'd welcome alternative suggestions that avoid this problem.